### ISSUE TITLE
Mac cmake build failure attempting to open libj9ddr_misc29.dylib.dSYM

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#9620]
https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_mac_cm_Nightly/10
```
22:25:57  [100%] Generating /Users/jenkins/workspace/Build_JDK11_x86-64_mac_cm_Nightly/build/macosx-x86_64-normal-server-release/vm/ddr_info/sets/j9ddr.macros
22:25:57  [100%] Generating /Users/jenkins/workspace/Build_JDK11_x86-64_mac_cm_Nightly/build/macosx-x86_64-normal-server-release/vm/runtime/j9ddr.dat, /Users/jenkins/workspace/Build_JDK11_x86-64_mac_cm_Nightly/build/macosx-x86_64-normal-server-release/vm/superset.dat
22:25:57  Error: /Users/jenkins/workspace/Build_JDK11_x86-64_mac_cm_Nightly/omr/ddr/lib/ddr-scanner/dwarf/DwarfScanner.cpp:1610 scanFile - Failure attempting to open /Users/jenkins/workspace/Build_JDK11_x86-64_mac_cm_Nightly/build/macosx-x86_64-normal-server-release/vm/runtime/libj9ddr_misc29.dylib.dSYM
22:25:57  Exiting...
```

**Comment by pshipton** [comment]
@dnakamura @keithc-ca 

Restarted the job to see if it's intermittent.
https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_mac_cm_Nightly/11/

**Comment by dnakamura** [comment]
Given the SHAs, I suspect eclipse/omr#5176 although I can't see any obvious reason why it would be failing

**Comment by pshipton** [comment]
The restart passed.

**Comment by keithc-ca** [comment]
I managed to reproduce this locally: Investigating.

**Comment by dnakamura** [comment]
I also managed to reproduce. Looked into it and it appears that we are no longer creating flattened debug info which means the debug info that ddrgen is attempting to open is a folder and not a file. Not sure why the restart passed, based on the issue it should be happening consistently. Currently have a fix pending in omr

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_mac_cm_Nightly/12

**Comment by keithc-ca** [comment]
See eclipse/omr#5199.

**Comment by keithc-ca** [comment]
This was fixed via eclipse/omr#5199.
See https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_mac_cm_Nightly/16.
